adopt-page.html

```diff
@@ -0,0 +1,193 @@
+          </p>
+          <a href="adopt-page.html">Adopt</a>
+        </div>
+        <div class="profile-card">
+          <img src="./images/violet-card.jpg" alt="Shot from above of a dark brown dog">
+          <h3>Violet</h3>
+          <p>
+            <span>Cost to Adopt:</span>
+            $123.45
+          </p>
+          <p>
+            Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae
+          </p>
+          <a href="adopt-page.html">Adopt</a>
+        </div>
+        <div class="profile-card">
+          <img src="./images/piper-card.jpg" alt="Front shot of a brown, smiling dog">
+          <h3>Piper</h3>
+          <p>
+            <span>Cost to Adopt:</span>
+            $123.45
+          </p>
+          <p>
+            Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae
+          </p>
+          <a href="adopt-page.html">Adopt</a>
+        </div>
+        <div class="profile-card">
+          <img src="./images/maximus-card.jpg" alt="Front shot of a black dog with a white nose">
+          <h3>Maximus</h3>
+          <p>
+            <span>Cost to Adopt:</span>
+            $123.45
+          </p>
+          <p>
+            Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae
+          </p>
+          <a href="adopt-page.html">Adopt</a>
+        </div>
+        <div class="profile-card">
+          <img src="./images/luna-card.jpg" alt="Facial shot of a tan dog">
+          <h3>Luna</h3>
+          <p>
+            <span>Cost to Adopt:</span>
+            $123.45
+          </p>
+          <p>
+            Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae
+          </p>
+          <a href="adopt-page.html">Adopt</a>
+        </div>
+        <div class="profile-card">
+          <img src="./images/stella-card.jpg" alt="Side shot of an orange dog lying down">
+          <h3>Stella</h3>
+          <p>
+            <span>Cost to Adopt:</span>
+            $123.45
+          </p>
+          <p>
+            Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae
+          </p>
+          <a href="adopt-page.html">Adopt</a>
+        </div>
+      </section>
+    </main>
+
+    <footer>
+      <img src="./images/adoptadog-logo.svg" alt="White outline of a dog sitting on its hind legs">
+      <p>
+        Aborpos nisto inci tet aut que volupti beruptiberro to ex esseque invenienihil imaximu scideru ptistiusam consequunt
+      </p>
+      <nav>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="adopt-page.html">Dogs</a></li>
+          <li><a href="blog-page.html">Blog</a></li>
+        </ul>
+      </nav>
+      <p>Copyright 2018 Adoptadop, LLC | Seattle, WA</p>
```

Comment:
you can use the copyright HTML symbol. You can google how to use HTML symbols (it's easy!)